target/linux/mvebu/base-files/lib/upgrade/clearfog.sh

```diff
@@ -18,8 +18,21 @@ platform_check_image_clearfog() {
 }
 
 platform_do_upgrade_clearfog() {
+	. /lib/functions.sh
```

Comment:
You can drop this line. `/lib/functions.sh` is included before any of the scripts in `/lib/upgrade/` is executed.

Reply:
ACK